src/wx/cmdevents.cpp

```diff
@@ -2872,6 +2872,11 @@ EVT_HANDLER(wxID_ABOUT, "About...")
 EVT_HANDLER(Bilinear, "Use bilinear filter with 3d renderer")
 {
     GetMenuOptionBool("Bilinear", gopts.bilinear);
+    // Force new panel with new bilinear option
+    if (panel->panel) {
```

Comment:
It's not necessary to check if the option has changed, because this code is handling a change event.